app/mailers/notification_mailer.rb

```diff
@@ -101,4 +115,8 @@ def global_validity_message(payload)
     def standard_global_validity_message(payload)
       "There #{payload[:error_count] == 1 ? 'is' : 'are'} #{pluralize(payload[:error_count], 'invalid record')}."
     end
+
+    def enable_settings_link
```

Comment:
this was just an improvement to move this logic from RadMailer into this subclass where it belongs more naturally